drivers/wifi/nrf700x/zephyr/src/zephyr_net_if.c

```diff
@@ -561,6 +587,11 @@ int wifi_nrf_if_stop_zep(const struct device *dev)
 	}
 
 	vif_ctx_zep->if_op_state = WIFI_NRF_FMAC_IF_OP_STATE_DOWN;
+
+	/* Remove the default VIF */
+	if (wifi_nrf_fmac_get_num_vifs(rpu_ctx_zep->rpu_ctx) == 0) {
+		wifi_nrf_fmac_dev_rem(rpu_ctx_zep->rpu_ctx);
```

Comment:
This crucial part is never reached, as we don't do `del_vif` in stop, so, no power savings. So, planning to move the all functionality that communicates with RPU from init to start/stop, that way this code will be reachable.